keyboards/sofle/keymaps/keyhive_via/via_layout.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
This is a VIA json, and belongs to the `the-via` repo, not here.  This needs to be removed.